packages/client/test/unit/setup.ts

```diff
@@ -10,20 +9,13 @@ import { dataUnionTemplate as templateJson, dataUnionFactory as factoryJson } fr
 import type { DataUnionTemplate, DataUnionFactory } from '@dataunions/contracts/typechain'
 
 import debug from 'debug'
-import { waitForCondition } from 'streamr-test-utils'
 const log = debug('DataUnionClient:unit-tests:withdraw')
 
 const ethereumRpcPort = Number.parseInt(process.env.GANACHE_PORT || "3456")
-const ethereumRpcServer = ganache.server({
-    // options, see https://github.com/trufflesuite/ganache/tree/develop/src/packages/ganache#startup-options
-    wallet: {
-        mnemonic: "testrpc"
-    },
-    logging: {
-        quiet: true // if only I could make it log to debug, not stdout...
-    }
-})
+const ethereumUrl = `http://localhost:${ethereumRpcPort}`
+const provider = new JsonRpcProvider(ethereumUrl)
 
+// start ganache with `npx ganache -p 3456 -m testrpc &`
```

Comment:
This comments seems that it's not in the right place?

Reply:
it's mostly the mnemonic. I'll clarify.